pod/perlvar.pod

```diff
@@ -578,7 +578,7 @@ Then
 
 would allocate a 64K buffer for use in an emergency.  See the
 L<INSTALL> file in the Perl distribution for information on how to
-add custom C compilation flags when compiling perl.  To discourage casual
```

Comment:
This is definitely talking about compiling the interpreter, not about compiling Perl code using the interpreter. The existing capitalization should stay.

Reply:
yeah, then it should be written as compiling the Perl interpreter?